typings/blockly.d.ts

```diff
@@ -1094,18 +1094,25 @@ declare module Blockly.Events {
     
             /**
              * Abstract class for a block event.
-             * @param {Blockly.Block} block The block this event corresponds to.
+             * @param {!Blockly.Block=} opt_block The block this event corresponds to.
```

Comment:
I would treat this file the same as blockly_compressed.js and not check it in, it will get rebuilt before release. (unless I'm missing some reason for this to be here)?